docs/root/start/install.rst

```diff
@@ -135,10 +135,10 @@ Install Envoy on Windows
 
 You can run Envoy using the official Windows development Docker image.
```

Comment:
Remove word "development"

Reply:
fixed in 8fc2acd074520e5aaf34b60591e715fc29db385e